docs/tutorial/install.rst

```diff
@@ -31,7 +32,7 @@ Then, let's install and configure the EdgeDB server:
 .. code-block:: bash
 
     $ edgedb server install
-    $ edgedb server init
+    $ edgedb server init default
```

Comment:
Perhaps use something like `myapp` here to highlight that the choice of the instance name is arbitrary and up to the user.  `default` might be interpreted like something you _have_ to use, or even as some weird argument to `init` ("init with defaults or something").